### ISSUE TITLE
DataGridView's edit controls don't support UIA Text pattern as required by UIA rules

### ISSUE THREAD

**Issue by merriemcgaw** [dotnet/winforms#2786]
* .NET Core Version: 
All

* Have you experienced this same bug with .NET Framework?: 
yes

**Problem description:**
DataGridView needs to support the TextPattern. Accessibility Insights reports failures for this issue

**Expected behavior:**
Accessibility Insights should not report failures.

**Minimal repro:**
Create a DataGridView with multiple data cells

**Comment by vladimir-krestov** [comment]
Original Bug [1159872](https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1159872)
Related to #2701

**Comment by vladimir-krestov** [comment]
Resolved by #3866

**Comment by Zheng-Li01** [comment]
Verified the issue with .NET SDK 5.0.100-rtm.20513.7 that the issue has been fixed in Edit Mode that the DataGridView's edit contorl is support Text Patter for now.

And Have filed a GitHub issue: [4031](https://github.com/dotnet/winforms/issues/4031)
